guidelines/content-guidelines/09-add-documentation-topic.md

```diff
@@ -0,0 +1,88 @@
+  description: "Overall documentation for API Gateway v2"
+  sources:
+    - type: markdown
+      name: docs
+      mode: package
+      url: https://github.com/{{ .Values.global.kymaOrgName }}/kyma/archive/{{ .Values.global.docs.clusterDocsTopicsVersion }}.zip
+      filter: /docs/api-gateway-v2/
+```
+​
+3. Adjust values for these fields:
+​
+- **cms.kyma-project.io/order** defines the order for your topic in the left navigation, such as `"11"`. To add it correctly, check in other `yaml` files which number is assigned to the last topic in the documentation navigation on the website, and add a consecutive number to your component. If you decide to modify the existing topic order, change the values for this parameter in all other `yaml` files accordingly to avoid duplicates.
+- **metadata.name** provides the name of the CR, such as `api-gateway-v2`.
+- **spec.displayname** defines the component name to show on the website, such as `"API Gateway v2"`.
+- **spec.sources.filter** defines the location of document sources for the new topic, such as `/docs/api-gateway-v2/`.
+​
+4. Merge the changes and wait until the website is rebuilt.
+​
+## Add a single OpenAPI specification
+​
+Apart from the generic Kyma documentation, there are also [OpenAPI](https://swagger.io/specification/) specifications rendered on the [Kyma website](https://kyma-project.io). These specifications show on the website under the **API Consoles** type in the right navigation panel of a given component topic.
+​
+To add a new specification, follow these steps:
+​
+1. Go to the [`templates`](https://github.com/kyma-project/kyma/tree/master/resources/core/charts/docs/charts/content-ui/templates) folder and locate the existing ClusterDocsTopic CR that you want to modify.
+​
+2. Add a new source entry in the **sources** field:
+​
+``` yaml
+sources:
+  ...
+  - type: {SPECIFICATION_TYPE}
+    name: {SPECIFICATION_NAME}
+    mode: single
+    url: {SPECIFICATION_URL}
+```
+​
+where:
+​
+- **{SPECIFICATION_TYPE}** defines a type of a given specification. Currently, only [OpenAPI](https://swagger.io/specification/) specifications are supported and they are defined under the `openapi` type.
+​
+- **{SPECIFICATION_NAME}** defines a unique identifier of a given specification. This field defines the URL on https://kyma-project.io/docs under which the specification is displayed. For example, if the specification is added in the `application-connector` ClusterDocsTopic CR with the `connectorapi` value of the **name** field, its URL is `https://kyma-project.io/docs/{VERSION_OF_DOCS}/components/application-connector/specifications/connectorapi/`.
+​
+- **{SPECIFICATION_URL}** defines the location of the specification. It may contain directives using values defined in `values.yaml` files. For internal specifications, defined in the [`kyma`](https://github.com/kyma-project/kyma) repository, it is recommended to use the directive with the Kyma version and the organization name, such as:
```

Comment:
```suggestion
- **{SPECIFICATION_URL}** defines the location of the specification. It may contain directives using values defined in `values.yaml` files. For internal specifications defined in the [`kyma`](https://github.com/kyma-project/kyma) repository, it is recommended to use the directive with a Kyma version and the organization name, such as:
```
I cannot find the Kyma version in the example... is it somewhere there?